pygbm/loss.py

```diff
@@ -104,5 +129,67 @@ def _update_gradients_hessians_logistic(gradients, hessians, y_true, y_pred):
             hessians[i] = gradient_abs * (1. - gradient_abs)
 
 
+class CategoricalCrossEntropy(Loss):
+
+    hessian_is_constant = False
+
+    def __call__(self, y_true, y_pred, average=True):
+        y_pred = y_pred.reshape(self.n_trees_per_iteration_, -1).T
```

Comment:
Actually, I think we should get rid off the `self.n_trees_per_iteration_` attribute completely in the loss functions. Here we could just do:

```python
 y_pred = y_pred.reshape(y_true.shape[0], -1)
```

(I think).